File: Serene/Serene.Script/Meeting/MeetingLocation/MeetingLocationDialog.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System.Collections.Generic;

    [IdProperty("MeetingLocationId"), NameProperty("Name"), IsActiveProperty("IsActive")]
    [FormKey("Meeting.MeetingLocation"), LocalTextPrefix("Meeting.MeetingLocation"), Service("Meeting/MeetingLocation")]
    public class MeetingLocationDialog : EntityDialog<MeetingLocationRow>
    {
    }
}

File: Serene/Serene.AspNetCore/Modules/Meeting/MeetingLocation/MeetingLocationDialog.ts

namespace Serene.Meeting {

    @Serenity.Decorators.registerClass()
    @Serenity.Decorators.responsive()
    export class MeetingLocationDialog extends Serenity.EntityDialog<MeetingLocationRow, any> {
        protected getFormKey() { return MeetingLocationForm.formKey; }
        protected getIdProperty() { return MeetingLocationRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingLocationRow.localTextPrefix; }
        protected getNameProperty() { return MeetingLocationRow.nameProperty; }
        protected getService() { return MeetingLocationService.baseUrl; }

        protected form = new MeetingLocationForm(this.idPrefix);

    }
}